test/unit/config.test.js

```diff
@@ -397,6 +131,7 @@ describe('config', function () {
       assert.notExists(config.httpOptions.httpRequestOptions);
     });
 
+    // TODO: do we care to cover this in a test? the code is pretty simple
```

Comment:
@vdiez this and the test above are about user error. Currently, our implementation will throw an error that the user will see. Is the intention that we should not fail and instead try to continue without informing the user of the misconfiguration?

Reply:
it was mostly about not failing with user env was "dirty", as supporting environment is never explicit of what the user want to be used as a param or not, so we were just proceeding anyway. I'm ok with throwing with a clear message

Reply:
I think this test was written to cover these lines
https://github.com/SonarSource/sonar-scanner-npm/blob/master/src/config.js#L127

I'm thinking that for server version checks, it might be a common place where users misconfigure their hostUrl and we could provide a better error message.